src/lang/en.h

```diff
@@ -283,8 +283,8 @@ const char *STR_EN[] = { ""
     /* 52 */ , "Ain't nothin' personal."
     /* 53 */ , "I still git a pain in my brain from ye.@An' it's tellin' me funny ideas now.@Like to shoot you to hell!"
     /* 54 */ , "You can't bump off me and my brood so easy, Lara."
-    /* 55 */ , "A leetle late for the prize giving - non?@Still, it is the taking-part wheech counts."
-    /* 56 */ , "You firin' at me?@You firin' at me, huh?@Ain't nobody else, so you must be firin' at me!"
+    /* 55 */ , "A leetle late for ze prize giving - non?@Still, it is ze taking-part wheech counts."
```

Comment:
I'm fairly certain Pierre says "A little late for ze prize giving - no?"

Reply:
The "non" is a tough one. Pierre obviously speaks in a heavily French-accented English, e.g. the "little"/"leetle" and his "the" definitely starts with a Z. The complicated part with "non" is that in French, the second N in "non" is silent. So a French guy saying "non" in French vs. saying "no" with a French accent is practically identical.

I think whoever made the subtitles originally did a decent job making the subtitles read as French-accented, and "non" is a good way to represent that. "Leetle" and "wheech" are maybe a bit weaker because a non-English speaker might not be able to identify what words those are supposed to be. But "non", I personally think is good.